link/kprobe.go

```diff
@@ -489,3 +495,14 @@ func kretprobeBit() (uint64, error) {
 	})
 	return kprobeRetprobeBit.value, kprobeRetprobeBit.err
 }
+
+// kprobeToken creates the SYM[+offs] token for the tracefs api.
+func kprobeToken(args probeArgs) string {
+	po := fmt.Sprintf("%s", args.symbol)
```

Comment:
golangci-lint fails, `args.symbol` is already a string

Reply:
ok